Loadouts/blumsv_brit_l85_ddpm.hpp

```diff
@@ -0,0 +1,318 @@
+// Author: BWMF
+// Description: US: HK416 - Marine Pattern Desert
+
+#include "undef.hpp" // Reset defines
+
+// Rifle
+#define RIFLE "CUP_arifle_L85A2_G","CUP_arifle_L85A2_NG"
+#define RIFLE_MAG "30Rnd_556x45_Stanag_red:8","30Rnd_556x45_Stanag_Tracer_Red:2"
+#define RIFLE_ATTACHMENTS "CUP_acc_ANPEQ_2"
+#define ALT_OPTICS STANAG_OPTICS
```

Comment:
Oh my god did we really define this as STANG_OPTICS? I'll have to fix that at some point.